### PR TITLE
Release 2.1.0

### LINKED ISSUE #2869: Prepare release 2.1.0
- [x] Resolve release blockers
  - [x] #2850
  - [x] #2851
  - [x] #2874
  - [x] #2878
- ~~Release new SDK version~~ n/a, 1.3.0 seems fine
- [x] Correct SDK version pinned in `.sdk-release`
- [x] Check translations
  - [x] SDK
  - [x] Backend
  - [x] Frontend
  - [x] Formio
- [x] Bump API version number
  - [x] Version bump
  - [x] Regenerate API spec
  - [x] Update READMEs with release dates + links
- [x] Bump version number (including package-lock.json)
- [x] Update changelog

### PR DESCRIPTION
**Pull request by sergei-maertens**
Closes #2869

### CODE REVIEW

File: src/openforms/js/lang/formio/en.json [hunk 1]
```diff
@@ -1,4 +1,37 @@
 {
+  "select-key": "select",
```
Comment (sergei-maertens) [inline src/openforms/js/lang/formio/en.json:2]: @SilviaAmAm can you do an English language double check? :grimacing:

Comment (SilviaAmAm) [reply src/openforms/js/lang/formio/en.json:2]: LGTM!

File: CHANGELOG.rst [hunk 2]
```diff
@@ -2,19 +2,28 @@
...
-This release candidate has focused on stability issues compared to the previous alpha
-version and includes some new experimental features.
+   **Gers** *[Gers]• Gaaf/mooi/leuk/geweldig/tof/heel goed*
+
+   -- Rotterdams Woordenboek
+
+Upgrade procedure
+-----------------
+
+Ensure that your current version of Open Forms is at least version 2.0.2 before
```
Comment (joeribekker) [inline CHANGELOG.rst:17]: Doublecheck: Are we blocking upgrades from before 2.0.2 to 2.1.0?

**Review by joeribekker — APPROVED** [review]

File: CHANGELOG.rst (hunk 2 above)
Comment (sergei-maertens) [reply CHANGELOG.rst:17]: yes, 2.0.0 or 2.0.1 to 2.1.0 is not possible, see https://github.com/open-formulieren/open-forms/blob/master/src/openforms/upgrades/upgrade_paths.py#L84

**Pull request merged**